core/src/ancestor_hashes_service.rs

```diff
@@ -277,14 +327,7 @@ impl AncestorHashesService {
                 retryable_slots_sender,
             );
         }
-        time.stop();
-        if total_packets >= *max_packets {
-            if time.as_ms() > 1000 {
-                *max_packets = (*max_packets * 9) / 10;
-            } else {
-                *max_packets = (*max_packets * 10) / 9;
-            }
-        }
```

Comment:
This was actually copy-pasta'd from here: https://github.com/solana-labs/solana/blob/8d980f07ba902788c166c0798e0c9668b5681717/core/src/serve_repair.rs#L351-L357, might be worth refactoring there as well for consistency

Reply:
👍 Implemented in the new commit.